addPlayer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once 'Player.php';
+require_once 'pdo-connection.php';
+require_once 'PlayerManager.php';
+
+// Retrieve the form data
+$name = $_POST['name'];
+$attack = $_POST['attack'];
+$defence = $_POST['defence'];
+$club_id = $_POST['club_id'];
+$position_id = $_POST['position_id'];
+$image = $_POST['image'];
```

Comment:
If someone were to load this page without submitting the form, you would get some errors here.

Worth just putting these lines in an if statement like so:

```
if (isset($_POST['name']) {
    $name = $_POST['name'];
    $attack = $_POST['attack'];
    $defence = $_POST['defence'];
    $club_id = $_POST['club_id'];
    $position_id = $_POST['position_id'];
    $image = $_POST['image'];
}

```

This just double checks that the data exists in $_POST before you try and actually use it

Reply:
Got it! Amended